ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -838,22 +839,26 @@ def test_find_orphan_automember_rules(self, hostgroup1):
         hostgroup1.ensure_missing()
 
         # Test rebuild (is failing)
-        try:
-            api.Command['automember_rebuild'](type=u'hostgroup')
-        except errors.DatabaseError:
-            pass
-        else:
-            pytest.fail("automember_rebuild was not failing with "
-                        "an orphan automember rule")
-
-        # Find obsolete automember rules
-        result = api.Command['automember_find_orphans'](type=u'hostgroup')
-        assert result['count'] == 1
-
-        # Find and remove obsolete automember rules
-        result = api.Command['automember_find_orphans'](type=u'hostgroup',
-                                                        remove=True)
-        assert result['count'] == 1
+        # Only run this test for 389-ds older than 1.4.0.22 where unmembering
+        # feature was implemented: https://pagure.io/389-ds-base/issue/50077
+        # As of today (2019-04-08), 1.3.9 is not released with this change yet
+        if ds_is_older('1.4.0.22'):
```

Comment:
Does it makes sense to verify that ``api.Command['automember_rebuild'](type=u'hostgroup')`` does not fail with >= 1.4.0.22?

Reply:
I think this is required to make sure that there is really no orphan automember rule after skipping the removal test.